docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -99,13 +99,13 @@ def validate(self):
         if self.settings.os not in ["Linux", "FreeBSD", "Macos"]:
             raise ConanInvalidConfiguration(f"{self.ref} is not supported on {self.settings.os}.")
 
-    # if another tool than the compiler or autotools is required to build the project (pkgconf, bison, flex etc)
+    # if a tool other than the compiler or autotools is required to build the project (pkgconf, bison, flex etc)
     def build_requirements(self):
         # only if we have to call autoreconf
-        self.tool_requires("libtool/x.y.z")
+        self.tool_requires("libtool/2.4.7")
```

Comment:
While the versions should be always checked and updated, the `libtool` and `automake` versions change very rarely, so it does not hurt to include them for convenience.